docs/repository-openapi.json

```diff
@@ -2049,6 +2266,25 @@
       }
     }
   },
+  "servers": [
```

Comment:
The last step in the documentation generation must've failed:
```
	jq '. | del(.servers)' docs/repository-openapi.json \
		| sponge docs/repository-openapi.json
```

Reply:
Got an error at first since sponge wasn't installed, so that's probably it.